src/Microsoft.Android.Sdk.ILLink/PreserveLists/System.Runtime.InteropServices.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<linker>
+	<!-- Workaround for https://github.com/xamarin/xamarin-android/issues/6626, remove once https://github.com/dotnet/runtime/pull/68265 lands -->
```

Comment:
Why are these workarounds still necessary?  dotnet/runtime#68265 was merged 2 months ago; that fix be in .NET 7.

Reply:
The comment is misleading, it applies to a different issue. `System.Runtime.InteropServices.UnmanagedCallersOnlyAttribute` must be preserved here or we get build errors:

```
Xamarin.Android.Common.targets(1463,3): error XAGJS7009: System.InvalidOperationException: Unable to find the System.Runtime.InteropServices.UnmanagedCallersOnlyAttribute type
Xamarin.Android.Common.targets(1463,3): error XAGJS7009:    at Xamarin.Android.Tasks.MarshalMethodsAssemblyRewriter.GetUnmanagedCallersOnlyAttributeConstructor(DirectoryAssemblyResolver resolver)
Xamarin.Android.Common.targets(1463,3): error XAGJS7009:    at Xamarin.Android.Tasks.MarshalMethodsAssemblyRewriter.Rewrite(DirectoryAssemblyResolver resolver)
Xamarin.Android.Common.targets(1463,3): error XAGJS7009:    at Xamarin.Android.Tasks.GenerateJavaStubs.Run(DirectoryAssemblyResolver res)
Xamarin.Android.Common.targets(1463,3): error XAGJS7009:    at Xamarin.Android.Tasks.GenerateJavaStubs.RunTask()
Xamarin.Android.Common.targets(1463,3): error XAGJS7009:    at Microsoft.Android.Build.Tasks.AndroidTask.Execute()
```